apps/emqx_resource/i18n/emqx_resource_schema_i18n.conf

```diff
@@ -135,14 +135,14 @@ When disabled the messages are buffered in RAM only."""
     }
   }
 
-  async_inflight_window {
+  inflight_window {
```

Comment:
Note: I noticed the frontend seems to have some i18n config paths they expect to find translations in.

So maybe just confirm that the frontend items that refer to the inflight window when creating a bridge still show the label and tooltip correctly.

Reply:
I have checked the dashboard UI for clickhouse and influxdb. The clickhouse bridge have a field labeled "Inflight window" that is always present. The influxdb bridge still have a field called "Async inflight window" that is only present when async mode is activated. I have yet to investigate why that is but at least it looks like the inflight labels are present in the dashboard.